CMakeLists.txt

```
@@ -104,6 +104,14 @@ endif()
 option(ZSTD_FROM_INTERNET "Download and use libzstd from the Internet" ${ZSTD_FROM_INTERNET_DEFAULT})
 find_package(zstd 1.1.2 REQUIRED)
 
+option(REDIS_STORAGE_BACKEND "Enable redis secondary storage")
+if(REDIS_STORAGE_BACKEND)
+  find_package(PkgConfig)
+  pkg_check_modules(HIREDIS REQUIRED hiredis>=0.13.3)
+  include_directories(${HIREDIS_INCLUDEDIR})
```

The `_INCLUDEDIR` is not [documented](https://cmake.org/cmake/help/v3.4/module/FindPkgConfig.html). 
```suggestion
  include_directories(${HIREDIS_INCLUDE_DIRS})
```
(Also the [cmake developer guide](https://cmake.org/cmake/help/latest/manual/cmake-developer.7.html#standard-variable-names) recommends `_INCLUDE_DIRS`)

---

INCLUDEDIR and LIBDIR are documented (under pkg_check_modules), but we could change variables.
Just that we gave up on using find_package and vars, because the provided files were not correct anyway.

---

Changed to use the prefix instead, as per discussion above.